frontend/app/views/support/Event.scala

```diff
@@ -0,0 +1,33 @@
+package views.support
+
+import model.Eventbrite.EBEvent
+
+object Event {
+
+  val ImportantClassName = "event-trait--important"
+  val ModerateClassName = "event-trait--moderate"
+  val NoteWorthyClassName = "event-trait--noteworthy"
```

Comment:
Not a massive fan on CSS classes in Scala, I think we could do something nicer here